lib/src/migrators/module.dart

```diff
@@ -12,6 +12,7 @@ import 'package:args/args.dart';
 import 'package:sass/src/ast/sass.dart';
 
 import 'package:path/path.dart' as p;
+import 'package:sass_migrator/src/migrators/module/forward_type.dart';
```

Comment:
It's generally preferred to use relative imports for files within the same `lib/`. Also below.